apps/fluent-tester/src/E2E/Switch/specs/SwitchSpec.win32.ts

```diff
@@ -0,0 +1,62 @@
+import NavigateAppPage from '../../common/NavigateAppPage.win';
+import SwitchPageObject, { SwitchComponentSelector } from '../pages/SwitchPageObject';
+import { PAGE_TIMEOUT, BOOT_APP_TIMEOUT, Keys } from '../../common/consts';
+
+// Before testing begins, allow up to 60 seconds for app to open
+describe('Switch Testing Initialization', function () {
+  it('Wait for app load', () => {
+    NavigateAppPage.waitForPageDisplayed(BOOT_APP_TIMEOUT);
+    expect(NavigateAppPage.isPageLoaded()).toBeTruthy(NavigateAppPage.ERRORMESSAGE_APPLOAD);
+  });
+
+  it('Click and navigate to Switch test page', () => {
+    /* Scroll to component test page button in scrollview if not already visible*/
+    SwitchPageObject.scrollToComponentButton();
+    SwitchPageObject.waitForButtonDisplayed(PAGE_TIMEOUT);
+
+    /* Click on component button to navigate to test page */
+    NavigateAppPage.clickAndGoToSwitchPage();
+    SwitchPageObject.waitForPageDisplayed(PAGE_TIMEOUT);
+
+    expect(SwitchPageObject.isPageLoaded()).toBeTruthy(SwitchPageObject.ERRORMESSAGE_PAGELOAD);
+    expect(SwitchPageObject.didAssertPopup()).toBeFalsy(SwitchPageObject.ERRORMESSAGE_ASSERT); // Ensure no asserts popped up
+  });
+});
+
```

Comment:
Please add Accessibility Testing.

See Checkbox.win32.spec.ts as an example.

Should test:
Accessibility Label
Accessibility Role

Reply:
After this PR I'm going to make a accessibility PR where I'll add the accessibility tests a long with it in an attempt to help declutter this PR. 

Although, in order to implement the onChange E2E I had to add some accessibility props but those aren't fully implemented. They were just added simply for the onChange E2E tests. I can add a Accessibility Role test though since that is pretty much set. Would that be fine?

Reply:
Sounds good to me as long as we make sure the other accessibility testing gets in in a subsequent PR :)